pages/linux/netselect.md

```diff
@@ -0,0 +1,24 @@
+# netselect
+
+> Speed test for choosing a fast network server.
+> More information: <https://github.com/apenwarr/netselect> .
+
+- Choose the fastest server:
+
+`sudo netselect {{host_1}} {{host_2}}`
+
+- Display nameserver resolution and statistics:
+
+`sudo netselect -vv {{host_1}} {{host_2}}`
+
+- Define maximum TTL (time to live):
+
+`sudo netselect -m {{10}} {{host_1}} {{host_2}}`
+
+- Print this many fastest servers:
```

Comment:
This description is a little bit confusing, what does the option `-s` do here?

Reply:
`-s` option prints  "x" fastest servers amongst "n" servers tested. 

"x" here is the number entered with `-s`.